app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
If the admins should be able to edit it, then yes.  I think it is odd that we have a single admin group. Many places have multiple levels of administrators or admin sets, with different administrators per set.